src/officehours_ui/templates/registration/login.html

```diff
@@ -28,12 +28,17 @@
                         <input class="form-control" name="password" type="password" placeholder="Password" autofocus autocomplete="current-password" required id="password">
                     </div>
                 </div>
-                <button type="submit" class="btn btn-primary">Login</button>
+                <button type="submit" class="btn btn-primary" onclick="setLoggedInValue()">Login</button>
```

Comment:
This could also be an EventListener similar to above. We generally try to avoid inline events.